pkg/apis/node.harvesterhci.io/v1beta1/nodeconfig.go

```diff
@@ -22,12 +22,18 @@ type NodeConfig struct {
 }
 
 type NodeConfigSpec struct {
-	NTPConfig *NTPConfig `json:"ntpConfigs,omitempty"`
+	NTPConfig      *NTPConfig      `json:"ntpConfigs,omitempty"`
+	LonghornConfig *LonghornConfig `json:"longhornConfigs,omitempty"`
```

Comment:
Ha! I also thought `NTPConfigs` looked wrong, but because it was already there with an `s` I used `LonghornConfigs` for consistency with it :-)

I'm happy to change it to `LonghornConfig` without the `s`.

Reply:
Cool, updated!